### PR TITLE
UICONSET-159: Add Authorization Roles to the consortium manager

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UICONSET-159:](https://folio-org.atlassian.net/browse/UICONSET-159) Add Authorization Roles to the consortium manager
## Approach

https://github.com/folio-org/ui-consortia-settings/assets/116072773/032cdc43-417d-4ebd-ad7a-486b1734c1ed

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by alisher-epam** [comment]
@folio-org/eureka please review it.

File: src/routes/ConsortiumManager/settings/authorization-roles/AuthorizationRolesSettings.test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    expect(screen.getByText('RoleCreate')).toBeInTheDocument();
+  });
+
+  it('should display edit authorization roles page', async () => {
+    window.location.pathname = '/consortia-settings/authorization-roles/1/edit';
+    renderComponent();
+
+    expect(screen.getByText('RoleEdit')).toBeInTheDocument();
+  });
+});
```
Comment (usavkov-epam) [inline src/routes/ConsortiumManager/settings/authorization-roles/AuthorizationRolesSettings.test.js:71]:
```suggestion
});

```

File: src/routes/ConsortiumManager/settings/authorization-roles/AuthorizationRolesSettings.test.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const renderComponent = () => render(<AuthorizationRolesSettings />, { wrapper });
+
+describe('AuthorizationRolesSettings', () => {
+  beforeEach(() => {
+    useAuthorizationRoles.mockClear().mockReturnValue({
+      roles: [],
+      isLoading: false,
+      onSubmitSearch: jest.fn(),
+    });
+  });
```
Comment (usavkov-epam) [inline src/routes/ConsortiumManager/settings/authorization-roles/AuthorizationRolesSettings.test.js:48]:
```suggestion
  });

```

File: src/routes/ConsortiumManager/settings/authorization-roles/Capabilities/CapabilitiesCompareItem/CapabilitiesCompareItem.test.js [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
+  screen, 
+  logDOM,
+} from '@folio/jest-config-stripes/testing-library/react';
 import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
-import { render, screen, logDOM } from '@folio/jest-config-stripes/testing-library/react';
-
 import {
   useRoleCapabilities,
   useRoleCapabilitySets,
+  useAuthorizationRoles,
```
Comment (usavkov-epam) [inline src/routes/ConsortiumManager/settings/authorization-roles/Capabilities/CapabilitiesCompareItem/CapabilitiesCompareItem.test.js:12]:
```suggestion
  useAuthorizationRoles,
  useRoleCapabilities,
  useRoleCapabilitySets,
```

File: translations/ui-consortia-settings/en.json [hunk 4]
```diff
@@ -6,6 +6,8 @@
   "document.settings.title": "Consortium manager settings",
 
   "button.confirm": "Confirm",
+  "button.actions": "Actions",
+  "button.new": "New",
```
Comment (usavkov-epam) [inline translations/ui-consortia-settings/en.json:10]: https://github.com/folio-org/stripes-components/blob/master/translations/stripes-components/en.json#L104

**Review by usavkov-epam — APPROVED** [review]

**Review by UladzislauKutarkin — APPROVED** [review]

**Pull request merged**